public/app/views/pdf/_titlepage.html.erb

```diff
@@ -1,8 +1,11 @@
 <div class="title-page">
     <div class="logo"><%= image_tag( record.resolved_repository['image_url'] || asset_path("archivesspace.small.png") ) %></div>
     <div class="title-block">
+    <% if record.finding_aid['title'] %>    
         <h1 class="title"><%= record.finding_aid['title'] %></h1>
         <h2 class="subtitle"><%= record.finding_aid['subtitle'] %></h2>
+    <% else %>
+        <h1 class="title"><%== record.display_string %></h1>
```

Comment:
@cdibella I think this should be **<%=** record.display_string %> instead of **<%==** record.display_string %>

Reply:
Thanks! I made that change (and removed an unrelated change I was working on). If that's too messy, just let me know if it would be better to close this one and start over fresh.